packages/@ember/-internals/runtime/lib/compare.ts

```diff
@@ -36,9 +36,7 @@ type Compare = -1 | 0 | 1;
 //                                `._________`-.   `.   `.___
 //                                              SSt  `------'`
 function spaceship(a: number, b: number): Compare {
-  let diff = a - b;
-  // SAFETY: Number casts true into 1 and false into 0. Therefore, this must end up as one of the Compare values.
```

Comment:
The comment needs updating but needs to remain because this OS do an unsafe cast.

Reply:
Resolved via a5027fefd